### DISCUSSION TITLE
Multicore

### DISCUSSION THREAD

**Discussion by amalrasas** [ChampSim/ChampSim#437] [Q&A]
Hi, i need help in how to run the simulator using more than one core, i didn't find anything in the docs regarding multicore, can anyone help?

**Comment by ngober** [comment]
Is [this](https://champsim.github.io/ChampSim/master/Creating-a-configuration-file.html#multi-core-configurations) the page you were looking for?

Multicore simulations are enabled by configuring with

    { "num_cores": 2 }

Then, when running the simulation, you must specify one trace per core. The simulation will not run otherwise.

**Reply by amalrasas** [reply]
how to specify one trace per core?

**Reply by ngober** [reply]
When you specify traces on the command line, specify the same number of traces as configured cores.
